trials/main-nightly/definition.yml

```diff
@@ -0,0 +1,30 @@
+name: main-nightly
+version: 1
+tenant_id: 5d23383f-2acb-448e-8353-4b4573b82276
+ui_client_id: f352ce15-0142-4dfa-8e18-801ee6391557
+spring_profile: dev
+spring_config_label: main
+site_service:
+  image_name: ghcr.io/ndph-arts/site-service
+  image_tag: main-nightly
+practitioner_service:
+  image_name: ghcr.io/ndph-arts/practitioner-service
+  image_tag: main-nightly
+init_service:
+  image_name: ghcr.io/ndph-arts/init-service
+  image_tag: main-nightly
+role_service:
+  image_name: ghcr.io/ndph-arts/role-service
+  image_tag: main-nightly
+spring_cloud:
+  gateway_service:
+    image_name: ghcr.io/ndph-arts/gateway-server
+    image_tag: main-nightly
+  discovery_service:
+    image_name: ghcr.io/ndph-arts/discovery-server
+    image_tag: main-nightly
+  config_server:
+    image_name: ghcr.io/ndph-arts/config-server
+    image_tag: main-nightly
+    git_uri: https://github.com/NDPH-ARTS/mts-trial-deployment-config.git
+    search_paths: trials/qa1/services, trials/qa1/services/*
```

Comment:
main-nightly will be the trial name so only alpha numeric value.
so need to rename the trial name and dir: nightly, or mainnightly

also the search paths need to point to the new directory and not qa1

Reply:
Renamed